platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/merkle/map/internal/ExpectedFCMFamilyImpl.java

```diff
@@ -761,7 +762,7 @@ private boolean creatingOnExistingEntities(
                         expectedValue.getEntityType());
             } else {
                 logger.error(
-                        PERFORM_ON_CREATE_DELETE,
```

Comment:
It seems that this marker is used here because this condition is acceptable under certain conditions, as documented at:

https://github.com/hashgraph/hedera-services/blob/4db779426c45f1d84eae528a147351a53d4915f3/platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/merkle/map/internal/ExpectedFCMFamilyImpl.java#L94-L99

I'm wondering if your change would break those cases or not. In an ideal world, the code would probably branch to detect those conditions and emit a warning instead of an error, I guess.

Reply:
@cody-littley : In an above comment, I see you're going to try and fix any potential failures after the change is deployed, which is fine.

In that case, if we don't want anyone to use non-standard markers, such as the one quoted above, why don't we delete them from the code-base to discourage their usage?

Reply:
I responded to this in the `#logging` slack channel. Let's discuss there.